server/neptune/config.go

```diff
@@ -21,15 +21,23 @@ type ServerConfig struct {
 	Port    int
 }
 
+type TerraformConfig struct {
+	DefaultVersionStr      string
+	DefaultVersionFlagName string
+	DownloadURL            string
+	LogFilters             valid.TerraformLogFilters
```

Comment:
Hmm I'm curious if we will need global config in the temporal worker for any other use cases since we build the root configs in the gateway. If we don't, I don't see much value in adding global config dependency just to pass in the log filters imo. What do you think?

Reply:
Currently LogFilters are supported as a global configuration, so I'd vote for not creating another separate location for setting them. In general, I think we want to [move towards using global config](https://github.com/lyft/atlantis/pull/275#discussion_r924966748) whenever possible to be our source of truth.